io.catenax.vehicle.product_description/3.0.0/ProductDescription.ttl

```diff
@@ -0,0 +1,511 @@
+#######################################################################
+# Copyright (c) 2023 Robert Bosch GmbH
```

Comment:
copyright from 2022, 2023?
Especially because some copyrights of 2.0.0 are missing now.

Reply:
Hi Caroline, that's correct. The version 3.0 comes only from use case quality. Also everybody who is mentioned in the copyright header has to bring a signed compliance by design document.

Reply:
Proposal: 
# Copyright (c) 2022 Allgemeine Deutsche Automobil-Club (ADAC) e.V
# Copyright (c) 2022 Badische Anilin und Sodafabrik societates Europaea
# Copyright (c) 2022, 2023 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)
# Copyright (c) 2022, 2023 ZF Friedrichshafen AG

etc.